crates/dbsp/src/tutorial.rs

```diff
@@ -213,7 +219,7 @@
 //! ```
 //!
 //! The best way to feed the records into `input_handle` is to collect them into
-//! a `Vec<(Record, isize)>`, where `isize` is the Z-set weight.  All the
+//! a `Vec<(Record, i64)>`, where `i64` is the Z-set weight.  All the
```

Comment:
I think there's a Rust `type` for weights somewhere in DBSP:
```
src/algebra/zset.rs:pub type ZWeight = i64;
```
BTW: the same mistake seems to be present in crates/dbsp/README.md

Reply:
I'll change it to use ZWeight.

Nothing tests the example in README, so it doesn't compile. I think we should just delete it (perhaps link to one of the simple examples instead).

Reply:
Yes, let's link to an example.